providers/apollo/types.go

```diff
@@ -11,16 +11,14 @@ var (
 
 type ObjectType string
 
-// postSearchObjects represents the objects that uses the searching endpoint,
-// POST method for requesting records.
+// postSearchObjects represents the objects that uses the searching endpoint,POST method.
 var postSearchObjects = []ObjectType{ //nolint:gochecknoglobals
 	"mixed_people", "mixed_companies", "contacts",
-	"accounts", "emails_campaigns",
```

Comment:
Are we unable to support `emails_campaigns` anymore? Or was this accidentally removed?

Reply:
We can, I have moved this to the `getSearchObjects` Because it does not have the limitations as the others in here. Using get with it works fine. The name is now `emailer_campaigns` or I may had it as a typo before. But it's now `emailer_campaigns` from their docs.